hack/deploy.sh

```diff
@@ -153,6 +153,8 @@ done
 # compare initial cluster SCCs to be sure HCO deployment didn't introduce any change
 dump_sccs_after
 
+"${CMD}" patch hyperconverged -n ${HCO_NAMESPACE} ${HCO_RESOURCE_NAME} --type='json' -p='[{"op": "add", "path": "/spec/infra/nodePlacement/nodeSelector/nodeType", "value": "infra" }, {"op": "add", "path": "/spec/workloads/nodePlacement/nodeSelector/nodeType", "value": "workloads" }]'
```

Comment:
This is not working. I tested this version and it worked for me:
```
"${CMD}" patch hyperconverged -n ${HCO_NAMESPACE} ${HCO_RESOURCE_NAME} --type='json' -p='[{"op": "add", "path":"/spec/infra", "value": {"nodePlacement": {"nodeSelector": {"nodeType": "infra"}}}}, {"op": "add", "path": "/spec/workloads", "value": {"nodePlacement": {"nodeSelector": {"nodeType": "workloads"}}}}]'
```

Reply:
Done, thanks